File: SwiftAndObjectiveC/FirstTask/FirstTask/SecondViewController.m
//
//  SecondViewController.m
//  FirstTask
//
//  Created by Brunno Castigrini on 06/08/21.
//

#import "SecondViewController.h"

@interface SecondViewController ()

@end

@implementation SecondViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

- (IBAction)dimissButton:(id)sender {
    [self dismissViewControllerAnimated:YES completion:nil];
}
@end

File: SwiftAndObjectiveC/FirstTaskSwift/FirstTaskSwift/SecondViewController.swift
//
//  SecondViewController.swift
//  FirstTaskSwift
//
//  Created by Brunno Castigrini on 06/08/21.
//

import UIKit

class SecondViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
    }


    @IBAction func returnButton(_ sender: Any) {
        
        self.navigationController?.popViewController(animated: true)
        
    }
}
